### PR TITLE
Update helm yaml

### PR DESCRIPTION
**Pull request by Salonijain27**
Updates the :
values.yaml file to use : `LoadBalancer` inorder to get an external ip address, rapids version, cuda version
helm version to `0.2.0` in the Chart.yaml

### CODE REVIEW

File: rapidsai/Chart.yaml [hunk 1]
```diff
@@ -1,7 +1,7 @@
-apiVersion: v1
+apiVersion: v2
```
Comment (jacobtomlinson) [inline rapidsai/Chart.yaml]: I think this needs to remain as `v1`

File: rapidsai/Chart.yaml [hunk 2]
```diff
@@ -1,7 +1,7 @@
...
 name: rapidsai
-version: 0.1.0
-appVersion: 0.10
+version: 0.2.0
+appVersion: "0.20"
```
Comment (jacobtomlinson) [inline rapidsai/Chart.yaml]: This should match the RAPIDS version `0.17`

File: rapidsai/values.yaml [hunk 3]
```diff
@@ -6,15 +6,15 @@ dask:
...
       pullPolicy: IfNotPresent
       # See https://kubernetes.io/docs/tasks/configure-pod-container/pull-image-private-registry/
       pullSecrets:
       #  - name: regcred
     replicas: 1
-    serviceType: "ClusterIP"
+    # serviceType: "ClusterIP"
     # serviceType: "NodePort"
-    # serviceType: "LoadBalancer"
+    serviceType: "LoadBalancer"
```
Comment (jacobtomlinson) [inline rapidsai/values.yaml]: We explicitly chose to make `ClusterIP` the default for security purposes. Having `LoadBalancer` as the default can result in the service being exposed to the internet. Therefore we should endeavour to make things secure by default.

Choosing `LoadBalancer` will likely be the most common use case, so perhaps we need to improve the documentation around this.

**Review by jacobtomlinson — COMMENTED** [review]
Thanks for this @Salonijain27.

A few comments.

File: rapidsai/values.yaml (hunk 3 above)
Comment (Salonijain27) [reply rapidsai/values.yaml]: updating the rapids.ai/cloud documentation to pass in ` --set dask.jupyter.serviceType="LoadBalancer"` while installing helm charts

Comment (jacobtomlinson) [reply rapidsai/values.yaml]: Perfect thanks!

**Pull request merged**